src/Core/Requests/RequestSerializer.cs

```diff
@@ -41,10 +41,21 @@ protected RequestSerializer()
         /// <param name="entity">The entity.</param>
         /// <returns>The serialized entity</returns>
         public virtual string SerializeEntity(object obj)
+        {
+            return SerializeEntity(obj, true);
+        }
+
+        /// <summary>
+        /// Serializes the entity to JSON.
+        /// </summary>
+        /// <param name="entity">The entity.</param>
+        /// <param name="preformConversion">This by defines whether or not a data conversion is performed on the object so that the HubSpot "property-value" syntax will be used for serialization. If this parameter is set to false no data conversion is performed and a standard object serialization is performed. </param>
+        /// <returns>The serialized entity</returns>
+        public virtual string SerializeEntity(object obj, bool performConversion)
```

Comment:
**suggestion**: set default value to true to avoid breaking default behavior.

Reply:
I have already prevented breaking default behaviour but I did it using a method overload. I have no problem with your suggestion of making performConversion a defaulted parameter (I agree that it is a better coding style).

Reply:
If I use a defaulted parameter @nover I get a lot of these syntax errors in unit test code:
   CS0854 C# An expression tree may not contain a call or invocation that uses optional arguments
That is why I originally chose to use the overloaded method rather than a defaulted parameter to prevent breaking existing code.
Is it OK for me to leave the code with the overloaded method?

Reply:
>  C# An expression tree may not contain a call or invocation that uses optional arguments

Oh yes those dreaded bastards. yes, it's fine :)